### DISCUSSION TITLE
user claims: `data` (//max number of bytes)

### DISCUSSION THREAD

**Discussion by siredmar** [nats-io/jwt#213] [General]
Hi! I need some help on the user JWT claims. There is a field called data in the user claims along with payload and subs. Payload and Subs does make sense. However the comment for data is just // Max number of bytes https://github.com/nats-io/jwt/blob/457367366f2ff72a3fefeed38b56a1da281476fb/v2/account_claims.go#L48C3-L48C3
Max number of bytes per what? Can anyone enlighten me?

**Comment by aricart** [comment]
This is for the maximum amount data that a client would be allowed to send - this is not currently enforced by the server.

**Comment by siredmar** [comment]
@aricart what is the difference between data and payload then? Would it be like: payload: 10 bytes, data: 10k meaning that the client is only allowed to send 1000 messages?

**Reply by aricart** [reply]
Total data the client could send.  Payload is max message size
